os/options.go

```diff
@@ -16,7 +13,7 @@ const (
 	EnvKeyClusterAddr           = "AGENT_CLUSTER_ADDR"
 	EnvKeyAgentSecret           = "AGENT_SECRET"
 	EnvKeyAgentSecurityShutdown = "AGENT_SECRET_TIMEOUT"
-	EnvKeyCapHostManagement     = "CAP_HOST_MANAGEMENT"
+	//EnvKeyCapHostManagement     = "CAP_HOST_MANAGEMENT"  // deprecated and unused
```

Comment:
just remove those

Reply:
Agree, should be done in https://github.com/portainer/agent/pull/228